pipeline/workflow/gnomad_processing.py

```diff
@@ -19,85 +19,75 @@
 class GnomADTask(DefaultPipelineTask):
     def __init__(self, *args, **kwargs):
         super(GnomADTask, self).__init__(*args, **kwargs)
-        self.gnomAD_download_file = "gnomAD.tsv"
 
 
 class DownloadGnomADData(GnomADTask):
-    gnomAD_static_data_url = luigi.Parameter(default='https://brcaexchange.org/backend/downloads/gnomAD_static_download_10_02_2020.tsv',
-                                            description='URL to download static gnomAD data from')
+    gnomAD_v2_static_data_url = luigi.Parameter(default='https://brcaexchange.org/backend/downloads/gnomAD_v2_hg19_10_02_2020.tsv',
+                                            description='URL to download static gnomAD v2 data from')
+    gnomAD_v3_static_data_url = luigi.Parameter(default='https://brcaexchange.org/backend/downloads/gnomAD_v3_GRCh38_03_10_2021.tsv',
+                                            description='URL to download static gnomAD v3 data from')
 
     def output(self):
-        return luigi.LocalTarget(self.assays_dir + "/gnomAD.tsv")
+        return { "v2": luigi.LocalTarget(f"{self.gnomad_file_dir}/gnomAD_v2_hg19_10_02_2020.tsv"),
+                 "v3": luigi.LocalTarget(f"{self.gnomad_file_dir}/gnomAD_v3_GRCh38_03_10_2021.tsv")}
 
     def run(self):
-        data = pipeline_utils.urlopen_with_retry(self.gnomAD_static_data_url).read()
-        with open(self.output().path, "wb") as f:
+        data = pipeline_utils.urlopen_with_retry(self.gnomAD_v2_static_data_url).read()
+        with open(self.output()["v2"].path, "wb") as f:
             f.write(data)
 
-"""
-################
-NOTE:
-gnomAD rarely updates its dataset
-due to issues with consistency downloading the same gnomad data for each release,
-a static file can be reused until new data is available
-see previous task for static data download
-################
-
-
-class DownloadGnomADData(GnomADTask):
-    def output(self):
-        return luigi.LocalTarget(
-            os.path.join(self.gnomad_file_dir, self.gnomAD_download_file))
-
-    def run(self):
-        os.chdir(gnomAD_method_dir)
-
-        args = ["python", "download_gnomad_data.py", "-o", self.output().path, "-l", self.artifacts_dir + "/gnomAD_download.log"]
-
-        pipeline_utils.run_process(args)
-        pipeline_utils.check_file_for_contents(self.output().path)
-"""
+        data = pipeline_utils.urlopen_with_retry(self.gnomAD_v3_static_data_url).read()
+        with open(self.output()["v3"].path, "wb") as f:
+            f.write(data)
```

Comment:
this could probably also be extracted into as separate function to avoid some redundancy